src/map/packet_system.cpp

```diff
@@ -7982,7 +7984,7 @@ void SmallPacket0x106(map_session_data_t* const PSession, CCharEntity* const PCh
         PTarget->pushPacket(new CInventoryItemPacket(PBazaar->GetItem(SlotID), LOC_INVENTORY, SlotID));
         PTarget->pushPacket(new CInventoryFinishPacket());
 
-        DebugBazaarsFmt("Bazaar Interaction [Purchase Successful] - Buyer: {}, Seller: {}, Item: {}, Qty: {}", PChar->name, PTarget->name, PItem->getName(), Quantity);
+        ShowInfo("SmallPacket0x106: Player '%s' purchased %u of item '%s' (%u) from player '%s' (Bazaar) (Total: %u gil)", PChar->name, Quantity, PItem->getName(), PItem->getID(), PTarget->name, PriceWithTax);
```

Comment:
Would it be more useful to bin all of this, and just introduce an audit_bazaar SQL table, and then async write into it like we do with chat messages?

That aside, why the rollback away from the safer and less error-prone Fmt logger?

Also also, we don't have it written as a rule anywhere, but it makes sense that the default amount of logging isn't spamming regular player actions. All of that sort of _additional_ logging should be opt in. I like to think of the default user as someone playing with their friends, and then the enhanced case is servers with lots of players who want to flip additional switches to turn more server-y things on.

Reply:
I'd like to create more tables but this will need some discussion and I thought improving the existing logging first was a good approach. More than happy to look at this though. But I can replace with the fmt version for now.

The logging for vendor sales and trades are not opt-in, so it made sense to me that bazaar purchase shouldn't be either.

Reply:
For now, apply your new information to `DebugBazaarsFmt`, since that was only added a month ago: https://github.com/LandSandBoat/server/pull/6297

and I'll hook up the audit tables probably on Sunday - with truncation jobs in `dbtool` etc.